src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/slack/impl/SlackMessagingServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.slack.impl;
+
+import com.slack.api.Slack;
+import com.slack.api.methods.MethodsClient;
+import com.slack.api.methods.SlackApiException;
+import com.slack.api.methods.request.chat.ChatPostMessageRequest;
+import com.slack.api.methods.response.chat.ChatPostMessageResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.common.ApplicationLogger;
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.slack.SlackMessagingService;
+
+import java.io.IOException;
+import java.util.List;
+
+@Service
+public class SlackMessagingServiceImpl implements SlackMessagingService {
+
+    private final ApplicationLogger logger;
+
+    @Value("${SLACK_CHANNEL}")
+    private String slackChannel;
+
+    @Value("${SLACK_ACCESS_TOKEN}")
+    private String slackAccessToken;
+
+    @Value("${RUN_APP_IN_ERROR_MODE}")
+    private boolean inErrorMode;
+
+    @Autowired
+    public SlackMessagingServiceImpl(ApplicationLogger logger) {
+        this.logger = logger;
+    }
+
+    @Override
+    public void sendMessage(List<String> failedMessageIds) {
+
+        try {
+            String slackErrorMessage = buildMessage(failedMessageIds);
+            Slack slack = Slack.getInstance();
+            MethodsClient methods = slack.methods(slackAccessToken);
+            ChatPostMessageRequest request = ChatPostMessageRequest.builder()
+                    .channel(slackChannel)
+                    .text(slackErrorMessage)
+                    .build();
+
+            ChatPostMessageResponse response = postSlackMessage(methods, request);
+            if(response.isOk()) {
+                logger.info(String.format("Message sent to: %s", slackChannel));
+            } else {
+                logger.info(String.format("Error message sent but received response: %s", response.getError()));
```

Comment:
this error logging should be a logger.error

Reply:
Changed to error